packages/babel-helper-create-class-features-plugin/test/fixtures/plugin-proposal-private-methods/loose-true/output.js

```diff
@@ -1,4 +1,4 @@
-var _privateMethod = /*#__PURE__*/babelHelpers.classPrivateFieldLooseKey("privateMethod");
+var _privateMethod = /*#__PURE__*/Symbol("privateMethod");
```

Comment:
It looks like `/*#__PURE__*/` is no longer needed.
https://github.com/babel/babel/pull/15393/files#diff-328fd0199b81baa72df150181152575e627b62d082281c6b5ac8f0075a7174b6R88

Reply:
I'm just about to remove the PURE comment for the new `Symbol` output and wondered whether it is needed for the existing standard solution, which generates `new WeakMap()` also with a PURE annotation. Shouldn't optimizer tools like Uglify _know_ that creating a (built-in) `WeakMap` has no side-effects?

Reply:
I'm talking about output like this:
https://github.com/babel/babel/blob/main/packages/babel-helper-create-class-features-plugin/test/fixtures/plugin-proposal-private-methods/loose-false/output.js#L1